src/desktop/apps/experimental_auth_modal/components/App.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+
+import Button from 'reaction/Components/Buttons/Default'
+import Modal from 'reaction/Components/Modal/Modal'
+import ModalHeader from 'reaction/Components/ModalHeader'
+import Title from 'reaction/Components/Title'
+
+const ModalStyle = {
+  height: '500px',
+  width: '500px',
+}
+
+export default class ModalDemo extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = { isModalOpen: false }
+    this.openModal = this.openModal.bind(this)
+    this.closeModal = this.closeModal.bind(this)
+  }
+
+  openModal() {
```

Comment:
Maybe instead of this getting triggered by an onClick we use a mediator like we do in the current auth modal pattern.